src/Umbraco.Web.UI.Client/src/views/components/property/property-actions/umb-property-actions.html

```diff
@@ -2,7 +2,10 @@
     <button type="button" class="btn-reset umb-outline umb-property-actions__toggle" ng-click="vm.toggle()" localize="title" title="propertyActions_tooltipForPropertyActionsMenu"><i></i><i></i><i></i></button>
 
     <div class="umb-property-actions__menu" role="menu" ng-if="vm.isOpen" on-outside-click="vm.close()" on-close="vm.close()" deep-blur="vm.close()">
-        <button class="umb-property-actions__menu-open-toggle" ng-click="vm.close()" tabindex="-1"><i></i><i></i><i></i></button>
+        <button
+            class="umb-property-actions__menu-open-toggle"
+            ng-click="vm.close()" tabindex="-1"><i></i><i></i><i></i> HEP HEY
```

Comment:
You're just testing my review skills, right? 😉

Reply:
Yes! 😄 ... But now I can't remember where this would actually be used... Can't seem to find it 🤯 Did you happen to stumble upon a place where it would render "HEP HEY"? Then I'll replace this with the ellipsis as well 👍

Reply:
Figured it out - It's on the datatype label... Not sure when it will be used so instead of using the new ellipsis component I'll leave it as is for now with the text removed of course 😅